bookShop/src/app/modules/components/cart/cart-component/cart-component.component.ts

```diff
@@ -1,28 +1,55 @@
-import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  OnInit,
+  ViewChild,
+} from '@angular/core';
 import { CartService } from '../../../../core/services/cart.service';
 import { IBook } from '../../../../shared/models/BookModel';
+import { OrderByPipe } from '../../../../shared/pipes/order-by.pipe';
 
 @Component({
   selector: 'app-cart-component',
   templateUrl: './cart-component.component.html',
   styleUrls: ['./cart-component.component.scss'],
 })
 export class CartComponentComponent implements OnInit {
+  @ViewChild('select')
+  el!: ElementRef;
+
+  orderbypipe = new OrderByPipe();
```

Comment:
You have to use pipe in the template, not in the class.

Reply:
Corrected